biome.jsonc

```diff
@@ -9,7 +9,8 @@
       "*-min.js",
       "bootstrap*.js",
       "jquery*.js",
-      "kobo/apps/openrosa/apps/logger/fixtures/*"
+      "kobo/apps/openrosa/apps/logger/fixtures/*",
+      "kobo/apps/organizations/static/*"
```

Comment:
Where did this change come from? It doesn't seem related to the tasks unless I'm mistaken 🤞

Reply:
Biome was complaining about these files, making the PR fail. Kalvis said to do that change :)

Reply:
Acutally Kalvis said something otherwise after some time :D https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/Biome.20issues/near/570825

I will undo this change